### ISSUE TITLE
Suggestion related to scales

### ISSUE THREAD

**Issue by PrismaNotTerra** [YuriSizov/boscaceoil-blue#51]
I am aware that fixing the Minor scale not being the Minor scale is basically impossible without a lot of compatibility sacrifices. However, there might be another way to have a correct Minor scale.
"Scale: Major" could be replaced with "Scale: Maj/Min". While this is selected, the letter in the bottom right corner would change to show both the major and minor chord that the key signature aligns with, like "X Maj / Y Min", or if that's too long, something like "XM / Ym".
So, Scale: Major set to C could instead be listed as Scale: Maj/Min and C Maj / A Min, to show that the scale also works with A Minor.
As for the saving issue, it could just save as the major chord. So, Scale: Maj/Min set to C Maj / A Min would save identically to how the original Bosca Ceoil saved Scale: Major set to C. And because A Minor uses the same 7 notes as C Major, nothing would get erased. Basically, it'd just be a visual thing.

**Comment by YuriSizov** [comment]
Hey! That's an interesting idea, but I don't think a complex workaround is worth it at this point. While I'm sure it is very annoying, and has been very annoying for a long time, that some scales are incorrectly coded, this has been the status quo of the app for many years now. So it's not a pressing issue and can wait just a few more months until the work on Bosca 4.0 starts and all the breaking changes become possible.

In other words, no point for a clever (but potentially confusing) trick when a proper solution can be just around the corner.
